### PR TITLE
Fix for the InvalidParemeter OU error

### PR DESCRIPTION
**Pull request by ArielLima**
Look for the `LegalEntity ID` inside the `AccountClaim CR` instead of the `Account CR`.
[Ticket](https://issues.redhat.com/browse/OSD-4015)

### CODE REVIEW

File: pkg/controller/accountclaim/organizational_units.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func MoveAccountToOU(r *ReconcileAccountClaim, reqLogger logr.Logger, accountCla
 }
 
 // CreateOrFindOU will create or find an existing OU and return its ID
-func CreateOrFindOU(reqLogger logr.Logger, client awsclient.Client, account *awsv1alpha1.Account, friendlyOUName string, baseID string) (string, error) {
+func CreateOrFindOU(reqLogger logr.Logger, client awsclient.Client, accountclaim *awsv1alpha1.AccountClaim, friendlyOUName string, baseID string) (string, error) {
```
[the remark this answers is not in the document]
Comment (fahlmant) [reply pkg/controller/accountclaim/organizational_units.go:76]: 👍 to accountClaim

**Comment by rogbas** [comment]
LGTM, @ArielLima can you please address the typo so we can merge?

**Comment by ArielLima** [comment]
@rogbas Done

**Comment by fahlmant** [comment]
@rogbas Please merge
@fahlmant please look

**Pull request merged**